game/tile.js

```diff
@@ -143,6 +143,17 @@ class TileTransition {
     this.type = TileTransitionType.SPAWN;
   }
 
+  loadTile(tile) {
+    // Spawn the saved progress after refresh or disconnection.
+    this.oldValue = 0;
+    assert(this.type === TileTransitionType.EMPTY);
+    this.tile = tile;
+    this.target = tile;
+    this.newValue = tile.value;
+    this.tile.value = this.oldValue;
+    this.type = TileTransitionType.SPAWN;
+  }
+
```

Comment:
The above function loads each tile into the transition, and it is being called by loadGame() on board.js.